var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -365,6 +366,8 @@ def install(self, spec, prefix):
             return
 
         # Remove libraries that the release version does not support
+        if spec.satisfies('@1.69.0:'):
+            with_libs.remove('signals')
```

Comment:
but you can build `Signals2` from boost 1.39.0 onward:
https://www.boost.org/doc/libs/1_69_0/doc/html/signals2.html
https://www.boost.org/doc/libs/1_39_0/doc/html/signals2.html
https://www.boost.org/users/history/version_1_69_0.html